packages/react-components/react-carousel/library/etc/react-carousel.api.md

```diff
@@ -209,6 +209,7 @@ export type CarouselProps = ComponentProps<CarouselSlots> & {
     align?: 'center' | 'start' | 'end';
     activeIndex?: number;
     onActiveIndexChange?: EventHandler<CarouselIndexChangeData>;
+    onAutoplayIndexChange?: (ev: null, data: CarouselAutoplayIndexChangeData) => void;
```

Comment:
While it sucks, probably it would be easier to pass just `new Event()` instead of having two separate callbacks, so consumers will have a single source of truth